Podfile

```diff
@@ -3,19 +3,19 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 platform :ios, '7.0'
 
-link_with 'ComponentKitTests',
-          'ComponentKitApplicationTests',
-          'ComponentTextKitApplicationTests'
 pod 'ComponentKit', :path => '.'
-pod 'ComponentKitTestLib', :path => './ComponentKitTestLib'
-pod 'ComponentSnapshotTestCase', :path => './ComponentSnapshotTestCase'
-pod 'OCMock', '~> 2.2'
-
-  
-
-
-
-
 
+target 'ComponentKitTests' do
+	pod 'OCMock', '~> 3.2'
+	pod 'ComponentKit/ComponentKitTestHelpers', :path => '.'
+	pod 'ComponentKit/ComponentKitTestLib', :path => '.'
+end
 
+target 'ComponentKitApplicationTests' do
+	pod 'ComponentKit/ComponentKitTestHelpers', :path => '.'
+	pod 'ComponentKit/ComponentSnapshotTestCase', :path => '.'
+end
```

Comment:
Here's an example of subspec usage... The 'ComponentKitApplicationTests' target needs access to the ComponentSnapshotTestCase and ComponentKitTestHelpers libraries. Cocoapods will only link those libraries to the  'ComponentKitApplicationTests' target.